HdmiCec/HdmiCec.cpp

```diff
@@ -112,73 +112,67 @@ namespace WPEFramework
 //=========================================== HdmiCec cec msg Processor =========================================
        void HdmiCec::process (const ActiveSource &msg, const Header &header)
        {
-             printHeader(header);
-             LOGINFO("Command: ActiveSource %s : %s  : %s \n",GetOpName(msg.opCode()),msg.physicalAddress.name().c_str(),msg.physicalAddress.toString().c_str());
-             HdmiCec::_instance->addDevice(header.from.toInt());
+            printHeader(header);
+            LOGINFO("Command: ActiveSource %s : %s  : %s \n",GetOpName(msg.opCode()),msg.physicalAddress.name().c_str(),msg.physicalAddress.toString().c_str());
+            HdmiCec::_instance->addDevice(header.from.toInt());
        }
        void HdmiCec::process (const ImageViewOn &msg, const Header &header)
        {
-             printHeader(header);
-             LOGINFO("Command: ImageViewOn from %s\n", header.from.toString().c_str());
-             HdmiCec::_instance->addDevice(header.from.toInt());
+            printHeader(header);
```

Comment:
Please get rid of whitespace changes.

Reply:
I got rid of extra space, now they are aligned properly.

Reply:
please remove whitespace changes. These are extra changes that no one needs